.github/ISSUE_TEMPLATE/2-question.md

```diff
@@ -13,6 +13,6 @@ Welcome!  If you have a question you'd like to ask, you can do it here or on the
 mlpack mailing list; see also http://mlpack.org/help.html.
 
 If you're looking for how to get involved and contribute, there's no need to
-open an issue---you can see http://www.mlpack.org/involved.html instead.
+open an issue---you can see http://www.mlpack.org/community.html instead.
```

Comment:
Always a fan of using https, don't feel obligated to change that.

Reply:
Agreed, just forgot about it.  I changed http->https in many places.

Reply:
Umm, should https be applied here as well?